lib/plugins/Label_HX.ts

```diff
@@ -0,0 +1,66 @@
+import { DecoderPlugin } from '../DecoderPlugin';
+import { DecodeResult, Message, Options } from '../DecoderPluginInterface';
+import { CoordinateUtils } from '../utils/coordinate_utils';
+import { ResultFormatter } from '../utils/result_formatter';
+
+export class Label_HX extends DecoderPlugin {
+  name = 'label-hx';
+
+  qualifiers() { // eslint-disable-line class-methods-use-this
+    return {
+      labels: ['HX'],
+      preambles: ['RA FMT LOCATION', 'RA FMT 43'],
+    };
+  }
+
+  decode(message: Message, options: Options = {}) : DecodeResult {
+    const decodeResult = this.defaultResult();
+    decodeResult.decoder.name = this.name;
+    decodeResult.message = message;
+    decodeResult.formatted.description = 'Undelivered Uplink Report';
+
+    const parts = message.text.split(' ');
+
+    decodeResult.decoded = true;
+    if (parts[2] === "LOCATION") {
+        let latdir = parts[3].substring(0, 1);
+        let latdeg = Number(parts[3].substring(1, 3));
+        let latmin = Number(parts[3].substring(3, 7));
+        let londir = parts[4].substring(0, 1);
+        let londeg = Number(parts[4].substring(1, 4));
+        let lonmin = Number(parts[4].substring(4, 8));
+        decodeResult.raw.position = {
+            latitude: (latdeg + latmin/60).toFixed(3) * (latdir === 'N' ? 1 : -1),
+            longitude: (londeg + lonmin/60).toFixed(3) * (londir === 'E' ? 1 : -1),
```

Comment:
nit (doesn't need to change) - i don't think `toFixed(3)` is needed here

Reply:
the `/60` for latmin and lonmin gives a repeating decimal, it's either this or put the full precision decimal into the test

Reply:
my personal preference - let the number be what it is. You can always test with a .toFixed() so you don't have a long decimal in the test. If you'd rather it stay, just let me know and i'll merge

Reply:
it's not a huge imposition to put the full decimal into the test and it is probably the "right way" since toFixed actually returns a string and only coincidentally gets turned back into a number. should be good to go now.